pkg/operator2/configobservation/infrastructure/observe_apiserverurl.go

```diff
@@ -0,0 +1,52 @@
+package infrastructure
+
+import (
+	"fmt"
+	"net/url"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+
+	"github.com/openshift/cluster-authentication-operator/pkg/operator2/configobservation"
+)
+
+func ObserveAPIServerURL(genericlisters configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (ret map[string]interface{}, _ []error) {
+	loginURLPath := []string{configobservation.OAuthServerConfigPrefix, "oauthConfig", "loginURL"}
+	defer func() {
+		ret = configobserver.Pruned(ret, loginURLPath)
+	}()
+	listers := genericlisters.(configobservation.Listers)
+	errs := []error{}
+
+	currentLoginURL, _, err := unstructured.NestedString(existingConfig, loginURLPath...)
+	if err != nil {
+		return existingConfig, append(errs, err)
```

Comment:
should we `continue` and try to fix it?

Reply:
yes, I meant to do that in the previous run but I see I've forgotten, thanks for pointing that out :+1: